cobra/test/test_io/test_json.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+"""Test functionalities of json.py"""
+
+from __future__ import absolute_import
+
+from os.path import join
+from warnings import warn
+
+import pytest
+from cobra import io
```

Comment:
I think, it'd be better to import the `json` stdlib directly here and `import cobra.io as cio` in order to not confuse it with the `io` standard package.